### PR TITLE
utility function for rounding numbers

### PR DESCRIPTION
**Pull request by theblushingcrow**

### CODE REVIEW

File: utils/src/number/index.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/**
+ *
+ * @param num - the number to round
+ * @param precision - how many placed after the decimal point to retain (default: 2, i.e. to the hundredth decimal)
+ * @returns {number} - the rounded number
+ */
+export function roundToDecimal(num, precision) {
+    if (isNaN(num)) {
```
Comment (NetanelBasal) [inline utils/src/number/index.ts]: You can use our util - isNumber

File: utils/__tests__/number.spec.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+describe('Numbers', () => {
+
+    describe('roundToDecimal', () => {
+
+        it('should throw an error when 1st argument is missing', () => {
+            const missingFirstArgument = () => {
+                return roundToDecimal();
+            }
+            expect(missingFirstArgument).toThrow('value to round is not a number');
```
Comment (NetanelBasal) [inline utils/__tests__/number.spec.ts]: I think you could do just:
`expect(roundToDecimal).toThrow('value to round is not a number');`

File: utils/__tests__/number.spec.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+        it('should round the 1st argument to 2 places after the decimal point when the 2nd argument is missing', () => {
+            expect(roundToDecimal(10)).toEqual(10);
+            expect(roundToDecimal(17.17)).toEqual(17.17);
+            expect(roundToDecimal(3.14159265358979323846264)).toEqual(3.14);
+        });
+
+        it('should override the rounding precision when 2nd argument is present', () => {
+            expect(roundToDecimal(10, 5)).toEqual(10);
+            expect(roundToDecimal(17.17, 1)).toEqual(17.2);
```
Comment (NetanelBasal) [inline utils/__tests__/number.spec.ts]: `expect(roundToDecimal(17.17, 1)).toEqual(17.2);`
Do you think it's a valid result?

Comment (theblushingcrow) [reply utils/__tests__/number.spec.ts]: I think so. 17.17 rounded to the nearest tenth is 17.2. Did you expect a different result?

File: utils/__tests__/number.spec.ts (hunk 2 above)
Comment (theblushingcrow) [reply utils/__tests__/number.spec.ts]: right, replaced line.

File: utils/src/number/index.ts (hunk 1 above)
Comment (theblushingcrow) [reply utils/src/number/index.ts]: done. isNumber is now a public validator function.

**Pull request merged**